### ISSUE TITLE
Evaluation results of SAVI pre-trained weights

### ISSUE THREAD

**Issue by sivannavis** [facebookresearch/sound-spaces#91]
Another question is about the results of evaluation on pretrained weights of SAVI:
My results look like this

> 2022-08-31 15:27:34,986 Average episode distance_to_goal: 10.023000
> 2022-08-31 15:27:34,986 Average episode normalized_distance_to_goal: 0.416810
> 2022-08-31 15:27:34,987 Average episode success: 0.233000
> 2022-08-31 15:27:34,987 Average episode spl: 0.159179
> 2022-08-31 15:27:34,987 Average episode softspl: 0.337451
> 2022-08-31 15:27:34,987 Average episode na: 178.965000
> 2022-08-31 15:27:34,987 Average episode sna: 0.115096
> 2022-08-31 15:27:34,987 Average episode sws: 0.147000
> 100%|██████████| 1000/1000 [15:40<00:00,  1.06it/s]

Is this tested on the unheard sound only? If so, could you tell me why the results don't match exactly with the performance in the paper?
![image](https://user-images.githubusercontent.com/50509896/187770382-b765e852-e1c1-4140-ae55-e79bb53fb3b3.png)

**Comment by ChanganVR** [comment]
Hi @sivannavis this discrepancy came as a result of an upgrade to latest Habitat version. I haven't been able to pinpoint where the issue exactly was. To compare the performance with SAVi, you could use the result you got by evaulating the model on the latest code.

**Comment by sivannavis** [comment]
@ChanganVR I see. Thanks for the explanation!
